webCommander/centralHub/management/commands/app_updater.py

```diff
@@ -0,0 +1,53 @@
+from django.core.management.base import BaseCommand, CommandError
+from centralHub.models import TwitterUser, TwitterUserPosts, SpotifyArtistInfo, SpotifyAlbumTracking
+from centralHub.spotify.spotifyAPI import search_spotify_artist, get_artist_albums, get_spotify_artist_by_id
+from centralHub.xTwitter.twitterAPI import get_top_tweets_from_user, get_tweets_since_last, XAPIUsageExceeded
+from centralHub.slackbot.slackbot_api import post_slack_message
+from centralHub.views import insert_posts_to_x_db
+
+class Command(BaseCommand):
+    help = 'Updates all trackers for new data'
+
+    def handle(self, *args, **options):
+        alltweets = TwitterUser.objects.all()
+        allartists = SpotifyArtistInfo.objects.all()
+
+        albums = []
+
+        for twitter_user in alltweets:
+            try:
+                new_posts = get_tweets_since_last(twitter_user.twitter_user_id,twitter_user.twitter_last_post_id)
+                if 'data' in new_posts.keys():
+                    created_count= insert_posts_to_x_db(twitter_user, new_posts)
+                    if created_count >0:
+                        post_slack_message(f"User {twitter_user.twitter_name} posted {created_count} new messages!")
+                        self.stdout.write(self.style.SUCCESS(f"{created_count} new posts for user: '{twitter_user.twitter_name}'! Notifying..."))
+                else:
+                    self.stdout.write(self.style.ERROR(f'No new posts for user {twitter_user.twitter_name}!!'))
+
+            except XAPIUsageExceeded:
+                self.stdout.write(self.style.ERROR(f"API Usage exceeded to query user {twitter_user.twitter_name}: Could not check"))
+
+        for artist in allartists:
+            album_count = 0
+            spotify_artist_albums = get_artist_albums(artist.spotify_artist_id)
```

Comment:
do we need a little sleep here in case there are many calls to the spotify api in case this was a multi-user app?